assets/global.js

```diff
@@ -325,6 +323,10 @@ class MenuDrawer extends HTMLElement {
     if (detailsElement === this.mainDetailsToggle) {
       if(isOpen) event.preventDefault();
       isOpen ? this.closeMenuDrawer(event, summaryElement) : this.openMenuDrawer(summaryElement);
+
+      if (window.matchMedia('(max-width: 990px)')) {
```

Comment:
This will apply to non-mobile browsers too, right? How significant a change is this for desktop browsers or tablets?

Reply:
Yes. So I used `990px` as it's the threshold where we're triggering the menu drawer. So it made sense to apply this fix on anything under 990 👍